example/timer/odp_timer_simple.c

```diff
@@ -69,6 +69,7 @@ int main(int argc ODP_UNUSED, char *argv[] ODP_UNUSED)
 		ret += 1;
 		goto err_tp;
 	}
+	memset(&tparams, 0, sizeof(tparams));
```

Comment:
Isn't this really exposing the omission of an `odp_timer_pool_param_init()` API?

Reply:
Sure, though that API would have to zero out all parameters in light of #965 - since user has to choose which resolution he wants. I don't have a strong opinion here though. Petri might have more insight why that API is not there.

Reply:
Bill may know better, but I suspect `odp_timer_pool_create()` API was created before we started the practice of adding init functions for parameters. At least from API consistency point of view adding `odp_timer_pool_param_init()` could be beneficial . I'll add it the API todo-list.

For now this fix is needed.

Reply:
@matiaselo Yes, you are correct. The init for the `odp_xxx_param_init()` functions was identified later but I had though by now ODP had added them all. Glad to hear this will be on the to-do list as this one clearly was an omission to that practice.